libipl/p10/ipl0.C

```diff
@@ -277,6 +277,43 @@ static bool update_genesis_hwas_state(void)
 	return true;
 }
 
+/**
+ * @brief Wrapper function to execute continue mpipl on the pib
+ *
+ * @param[in] pib pib target to operate on
+ *
+ * return ipl error type enum
+ */
+static ipl_error_type  ipl_sbe_mpipl_continue(struct pdbg_target *pib)
+{
+	ipl_log(IPL_INFO, "Enter: ipl_sbe_mpipl_continue(%s)", pdbg_target_path(pib));
+
+	// Get the current SBE state
+	enum sbe_state state;
```

Comment:
Generally define variables before executable statement.  Makes them easier to find.